ravenpy/models/emulators.py

```diff
@@ -131,6 +135,17 @@ def derived_parameters(self):
                 index=sb.subbasin_id
             )
 
+        self.rvh.lake_subbasins = tuple(
+            [sb.subbasin_id for sb in self.rvh.subbasins if sb_is_lake[sb.subbasin_id]]
+        )
+        self.rvh.land_subbasins = tuple(
+            [
+                sb.subbasin_id
+                for sb in self.rvh.subbasins
+                if not sb_is_lake[sb.subbasin_id]
+            ]
+        )
+
```

Comment:
That's pretty awesome!!

Could you please then also add the following two lines below these `SubBasinGroup`'s?
```
:SBGroupPropertyMultiplier	Land	MANNINGS_N	            1.0
:SBGroupPropertyMultiplier	Lakes	RESERVOIR_CREST_WIDTH	1.0
```
These lines are the actual reason why we create those groups. Otherwise the groups are not used in the setup. Plus we will need these two lines when we start calibrating the GR4J+routing example. In calibration mode the two `1.0`'s will be converted into two parameters (e.g., `par_x08` and `par_x09`). Merci.

Reply:
done!